js/gate.js

```diff
@@ -3120,6 +3125,7 @@ module.exports = class gate extends Exchange {
             'margin': 'privateSpotPost' + methodTail,
             'swap': 'privateFuturesPostSettle' + methodTail,
             'future': 'privateDeliveryPostSettle' + methodTail,
+            'option': 'privateOptionsPost' + methodTail,
```

Comment:
Since I've used it for `fetchTicker` and `createOrder` right now, after doing a test I found some need for extra fix, mainly for `fetchTickers` and `fetchOrder`.

Here is the test code used.
```python

import pytest
from ccxt import gateio, gate

option_ticker = 'BTC/USDT:USDT-220729:22000:P'

@pytest.fixture
def ccxtga():
    credentials = {"apiKey":"","secret":""}
    ccxtga = gateio(credentials)
    ccxtga.load_markets(params={'type': 'option'})
    return ccxtga

def test_gateio_option_fetch_ticker(ccxtga: gate):
    assert ccxtga.fetch_ticker(option_ticker)['symbol'] == option_ticker

def test_gateio_option_fetch_tickers(ccxtga: gate):
    assert ccxtga.fetch_tickers([option_ticker])[option_ticker]['symbol'] == option_ticker

def test_gateio_option_fetch_trades(ccxtga: gate):
    assert ccxtga.fetch_trades(option_ticker)[0]['symbol'] == option_ticker

def test_gateio_option_fetch_my_trades(ccxtga: gate):
    assert len(ccxtga.fetch_my_trades(option_ticker)) == 0

def test_gateio_option_fetch_order(ccxtga: gate):
    assert ccxtga.fetch_order(None, option_ticker)['symbol'] == option_ticker

def test_gateio_option_fetch_orders_by_status(ccxtga: gate):
    assert ccxtga.fetch_orders_by_status(None, option_ticker)['symbol'] == option_ticker

# stateful endpoints below
def test_gateio_option_create_order(ccxtga: gate):
    # just to make sure that it calls correct API endpoint
    assert ccxtga.create_order(option_ticker, 'limit', 'buy', 0, 0)['symbol'] == option_ticker

def test_gateio_option_cancel_order(ccxtga: gate):
    # just to make sure that it calls correct API endpoint
    assert ccxtga.cancel_order(None)['symbol'] == option_ticker

def test_gateio_option_cancel_all_orders(ccxtga: gate):
    # just to make sure that it calls correct API endpoint
    assert ccxtga.cancel_all_orders(option_ticker)[0]['symbol'] == option_ticker
```

Will come back later with the fix.

Reply:
@susemeee thanks you, looking forward to it!